README.md

```diff
@@ -54,19 +54,19 @@ you must add the following dependencies:
 composer require symfony/dependency-injection symfony/http-kernel
 ```
 
-Enable the bundle on the `AppKernel` class:
+Register the bundle in the kernel of your application:
```

Comment:
Just like you did for Composer, BTW. Plus beginners will probably not begin by installing this library ;)